src/modules/ROOT/pages/changelog/202303/06.json

```diff
@@ -0,0 +1,118 @@
+{
+  "meta": {
+    "releasedOn": "2023.09.13"
+  },
+  "added": [
+    {
+      "what": [
+        "splice function in workflows that can change the contents of a collection of elements by removing or replacing existing elements and/or adding new elements in place."
+      ],
+      "why": [
+        "to provide the option to easily add/remove elements from a collection."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/7bc36eb46f57cf1d8a271319f04c8ed5158eb7a7"
+      ]
+    },
+    {
+      "what": [
+        "the option to fetch data for a geometry page block when clicking on a button in the page block or after a set by the user amount of time."
```

Comment:
I don't understand what this does. Is this for the auto-refresh? If so, let's use the same wording as we used back then please